wicket-core/src/main/java/org/apache/wicket/ajax/res/js/wicket-ajax-jquery.js

```diff
@@ -2116,10 +2116,15 @@
 
 			// Adds the element to page head
 			addElement: function (element) {
-				var head = document.getElementsByTagName("head");
+				var headItems = document.querySelector('head meta[name="wicket.header.items"]');
```

Comment:
Seems we haven't used querySelector() before - are we increasing the minimum supported browser version with this?

Reply:
According to this table [1] `querySelector` is being supported by all browsers ...

[1] https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector#Browser_Compatibility